scripts/migration/lib.sh

```diff
@@ -283,6 +283,44 @@ check_version() {
     fi
   fi
 
+  cluster_version="$(get_apps_version "${1}" >/dev/null 2>&1 || true)"
+  case "${2:-}" in
+  prepare)
+    # Ensure not in the middle of upgrading
+    if get_prepared_version "${1}" &>/dev/null; then
+      log_warn "Migration already in progress"
+      if [[ -t 1 ]]; then
+        log_warn_no_newline "Do you want to continue [y/N]: "
+        read -r reply
+        if [[ "${reply}" != "y" ]]; then
+          exit 1
+        fi
+      else
+        # Don't automatically proceed
+        exit 1
+      fi
+    fi
+
+    # Make sure **config** and **cluster** is on the same (old) version
+    if [[ -n "${cluster_version}" ]] && [[ "${cluster_version}" != "${VERSION["${1}-config"]%.*}" ]]; then
+      log_warn "Version mismatch, cluster ${cluster_version}, config ${VERSION["${1}-config"]%.*}"
```

Comment:
I'm still missing the warning message here for when there is no cluster version.

Reply:
Oh, thought I did that, where did that change go?

Oh, in `common.bash` https://github.com/elastisys/compliantkubernetes-apps/pull/2399/commits/4539515ef5ea206eed5d047bc31043cbd33f3149 I'll do it here too.

Reply:
How about d4bcbec9 ?